src/types/types.ts

```diff
@@ -150,11 +150,12 @@ export interface IDesign {
 
```

Comment:
let's decompose this file into different ones. 
types/
- case.ts
- trajectory.ts